controllers/virtinkmachine_controller.go

```diff
@@ -46,6 +46,7 @@ type VirtinkMachineReconciler struct {
 //+kubebuilder:rbac:groups=virt.virtink.smartx.com,resources=virtualmachines,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch
 //+kubebuilder:rbac:groups=cdi.kubevirt.io,resources=datavolumes,verbs=get;list;watch;create;update;patch;delete;
+//+kubebuilder:rbac:groups="",resources=persistentvolumeclaims,verbs=get;list;watch
```

Comment:
We can merge this line with line 47?

Reply:
yes we can, but we'd better not, keep it separate is more clear.